templates/chemicals/chemical_edit_form.html

```
@@ -15,15 +15,20 @@ <h4 class="modal-title">Edit {{ form.instance }}</h4>
             </div>
             <div class="modal-footer">
                 <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
-                <input type="submit" class="btn btn-primary" id="saveChem" value="Save changes" />
+                <input type="submit" class="btn btn-primary" id="saveChem" value="Save changes"/>
             </div>
         </div><!-- /.modal-content -->
     </form>
 </div><!-- /.modal-dialog -->
 <script>
     var form_options = {
         target: '#chemical-update-modal',
-        success: function (response) {}
+        success: function (response) {
+        },
+        error: function (response) {
+            alert(response.responseText);
+        }
```

An error when saving an edit from the modal window was failing silently. This causes the error to be displayed